src/Symfony/Component/Lock/Tests/Store/StoreFactoryTest.php

```diff
@@ -50,13 +51,25 @@ public function validConnections()
         }
         if (class_exists(\Zookeeper::class)) {
             yield [$this->createMock(\Zookeeper::class), ZookeeperStore::class];
+            yield ['zookeeper://localhost:2181', ZookeeperStore::class];
         }
         if (\extension_loaded('sysvsem')) {
             yield ['semaphore', SemaphoreStore::class];
         }
         if (class_exists(\Memcached::class) && class_exists(AbstractAdapter::class)) {
             yield ['memcached://server.com', MemcachedStore::class];
         }
+        if (class_exists(\Redis::class) && class_exists(AbstractAdapter::class)) {
+            yield ['redis://localhost', RedisStore::class];
+        }
+        if (class_exists(\PDO::class)) {
+            yield ['sqlite:/tmp/sqlite.db', PdoStore::class];
```

Comment:
Does it work when using `sqlite:///tmp/sqlite.db`, with DBAL?
If yes, let's add a few more URL-like DSNs next to PDO-only ones?

Reply:
no. It's addressed in https://github.com/symfony/symfony/pull/34057